flow/layers/image_filter_layer_unittests.cc

```diff
@@ -289,12 +289,20 @@ TEST_F(ImageFilterLayerTest, ChildIsCached) {
   EXPECT_EQ(raster_cache()->GetLayerCachedEntriesCount(), (size_t)0);
   EXPECT_FALSE(raster_cache()->Draw(mock_layer.get(), other_canvas));
   EXPECT_FALSE(raster_cache()->Draw(mock_layer.get(), cache_canvas));
+  EXPECT_FALSE(raster_cache()->Draw(layer.get(), other_canvas,
+                                    RasterCacheLayerStrategy::kLayerChildren));
+  EXPECT_FALSE(raster_cache()->Draw(layer.get(), cache_canvas,
+                                    RasterCacheLayerStrategy::kLayerChildren));
```

Comment:
Also EXPECT_FALSE on `Draw(layer, kLayer)`? (twice)

And FALSE/FALSE after the preroll below on the same pattern.

Reply:
(Same comment on all 3)

Reply:
Canvas for the two lines of code here are not same. The logic here is consistent with the previous logic.

Reply:
I am asking for additional tests. I understand the difference in the existing tests.

You test mock layer with both canvases
Then you test the parent layer with both canvases with kLayerChildren

I'm asking you to add 2 more lines that tests the parent layer with both canvases, but with kLayer specified.

Reply:
Ahh, I see. It's done by https://github.com/flutter/engine/pull/31875/commits/5f574d2b3d282a8056c05a93270a87ab2fb74ea3